tools/code_coverage/pull_request.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+
+"""
+usage: pull_request.py files/diff pull_id
+"""
+
+import argparse
+import os
+
+from github import Github
+
+# Retrieve the GitHub API token from environment variables
+token = os.getenv("GITHUB_API_TOKEN")
+
+
+def get_pull(pull_id):
+    github = Github(token, timeout=60)
+    repo = github.get_repo("FlagOpen/FlagGems")
+    return repo.get_pull(pull_id)
+
+
+def get_files(args):
+    pull = get_pull(args.pull_id)
+    for file in pull.get_files():
+        print(f"/work/FlagGems/{file.filename}")
```

Comment:
Do we have a workdir like env somewhere around?

Reply:
no, but in the ci machine it's `__w` dir，i'll fix that